### PR TITLE
Improve mailing lists description

### PR DESCRIPTION
**Pull request by DelazJ**
There were mailing lists without description. Picked text from http://docs.qgis.org/testing/en/docs/user_manual/preamble/help_and_support.html

### CODE REVIEW

File: source/site/getinvolved/mailinglists.rst [hunk 1]
```diff
@@ -38,35 +40,49 @@ http://lists.osgeo.org/mailman/listinfo/qgis-developer
...
 
 http://lists.osgeo.org/mailman/listinfo/qgis-community-team
 
 
 QGIS translation
 -----------------
 
-List for translators of either the website, the documentation or the QGIS itself
-
+This list deals with the translation efforts. If you like to work on the
```
Comment (yjacolin) [inline source/site/getinvolved/mailinglists.rst]: If you **want** to work on ...

File: source/site/getinvolved/mailinglists.rst [hunk 2]
```diff
@@ -38,35 +40,49 @@ http://lists.osgeo.org/mailman/listinfo/qgis-developer
...
+translation of the web site, the manuals or the graphical user interface (GUI),
```
Comment (yjacolin) [inline source/site/getinvolved/mailinglists.rst:55]: manual ?

Comment (DelazJ) [inline source/site/getinvolved/mailinglists.rst:55]: I think it's about user manual, training manual, Cookbook, Guidelines...

File: source/site/getinvolved/mailinglists.rst (hunk 1 above)
Comment (DelazJ) [inline source/site/getinvolved/mailinglists.rst]: done

File: source/site/getinvolved/mailinglists.rst (hunk 2 above)
Comment (yjacolin) [inline source/site/getinvolved/mailinglists.rst:55]: ok you're right :)

**Comment by yjacolin** [comment]
just merge it after the test check.

**Pull request merged**